knora/dsplib/schemas/data.xsd

```diff
@@ -424,6 +425,9 @@
         <xs:attribute name="label" type="xs:string" use="required"/>
         <xs:attribute name="id" type="xs:ID" use="required"/>
         <xs:attribute name="permissions" type="xs:NCName" use="optional"/>
+        <xs:attribute name="iri" type="xs:string" use="optional"/>
```

Comment:
IRI and ARK were, until now, not supported for <annotation>, <link>, and <region>. But I think they should.